designcompose/src/main/java/com/android/designcompose/DCTraces.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose
+
+object DCTraces {
+    const val DESIGNFRAME_DE_SUBSCRIBE = "DesignFrame DisposableEffect subscribe"
+    const val DESIGNFRAME_FINISHLAYOUT = "DesignFrame.finishLayout"
+    const val DESIGNTEXT_DE = "DesignText DisposableEffect"
```

Comment:
Seems a bit odd that this doesn't have "SUBSCRIBE" in the name but the DesignFrame one does.

Reply:
True